marvel-app/core/models/ComicDataWrapper.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ComicDataWrapper.swift
+//  marvel-app
+//
+//  Created by Michal on 01/08/2022.
+//
+
+import Foundation
+
+struct ComicDataWrapper: Codable {
+    public let code: Int?
```

Comment:
Dont add nill options to structs if they are not nullable. With bad decoding, you will never get an error but only an empty object.